common/djangoapps/student/reports.py

```diff
@@ -0,0 +1,69 @@
+import unicodecsv as csv
+
+from django.utils import timezone
+from xmodule.modulestore.exceptions import ItemNotFoundError
+
+from lms.djangoapps.grades.course_grade_factory import CourseGradeFactory
+
+from lms.djangoapps.grades.models import PersistentCourseGrade
+
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+def csv_course_report(f):
+    field_names = [
+        'Candidate RSA ID number',
```

Comment:
Not sure where you are getting this value from as it is not passed to edx from the enrollment API

Reply:
It seems that you are assigning the candidate GUID to the RSA ID number - this is not correct

Reply:
@nerfani it seems we misunderstood to source of data we can use to provide you with an `Candidate RSA ID number`.
Could you advice the correct place we the analytic report can get this data?
Thanks.